go/teams/treeloader.go

```diff
@@ -63,38 +63,43 @@ func (l *Treeloader) Load(mctx libkb.MetaContext) error {
 	l.targetTeamName = target.Name()
 
 	// Load rest of team tree asynchronously.
-	go func() {
+	go func(start time.Time, targetChainState *TeamSigChainState) {
```

Comment:
why are you passing start and targetChainState as parameters but not mctx (or better imctx) and l?

Reply:
I probably don't need to pass any of them since they're not being changed